tests/test_loaders.py

```diff
@@ -106,7 +106,7 @@ def test_dataset_attributes(httpserver):
             clean_remote_dataset(dataset_name)
 
 
-def test_cite(httpserver):
+def test_cite_license(httpserver):
```

Comment:
nit - `test_cite_and_license`

Reply:
it would also be good to add a line to `test_dataset_attributes` to check that dataset.license is set and not None